assets/js/common/ChartDisabledBox/ChartDisabledBox.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function ChartDisabledBox() {
```

Comment:
Maybe `ChartsDisabledBox` as it disables multiple charts (of course based on the usage) and the text says so

Reply:
done